modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -0,0 +1,106 @@
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in GCP.
+* Create and configure networking and load balancers in GCP.
+* Create control plane and compute roles.
+
+.Procedure
+
+. Copy the template from the *Deployment Manager template for the bootstrap machine*
+section of this topic and save it as `04_bootstrap.py` on your computer. This
+template describes the bootstrap machine that your cluster requires.
+
+. Export the following variables required by the resource definition:
++
+----
+$ export CONTROL_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --region=${REGION} --format json | jq -r .selfLink`
+$ export CLUSTER_IMAGE=`gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+$ export ZONE_0=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[0] | cut -d "/" -f9`
+$ export ZONE_1=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[1] | cut -d "/" -f9`
+$ export ZONE_2=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[2] | cut -d "/" -f9`
+----
+
+. Create a bucket and upload the `bootstrap.ign` file:
++
+----
+$ gsutil mb gs://${INFRA_ID}-bootstrap-ignition
+$ gsutil cp bootstrap.ign gs://${INFRA_ID}-bootstrap-ignition/
+----
+
+. Create a signed URL for the bootstrap instance to use to access the Ignition
+config. Export the URL from the output as a variable:
++
+----
+$ export BOOTSTRAP_IGN=`gsutil signurl -d 1h service-account-key.json \
+    gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign | grep "^gs:" | awk '{print $5}'`
+----
+
+. Create a `04_bootstrap.yaml` resource definition file:
++
+----
+$ cat <<EOF >04_bootstrap.yaml
+imports:
+- path: 04_bootstrap.py
+
+resources:
+- name: cluster-bootstrap
+  type: 04_bootstrap.py
+  properties:
+    infra_id: '${INFRA_ID}' <1>
+    region: '${REGION}' <2>
+    zone: '${ZONE_0}' <3>
+
+    cluster_network: '${CLUSTER_NETWORK}' <4>
+    control_subnet: '${CONTROL_SUBNET}' <5>
+    image: '${CLUSTER_IMAGE}' <6>
+    machine_type: 'n1-standard-4' <7>
+    root_volume_size: '128' <8>
+
+    bootstrap_ign: '${BOOTSTRAP_IGN}' <9>
+EOF
+----
+<1> `infra_id` is the `INFRA_ID` infrastructure name from the extraction step.
+<2> `region` is the region to deploy the cluster into, for example `us-east1`.
+<3> `zone` is the zone to deploy the bootstrap instance into, for example `us-east1-b`.
+<4> `cluster_network` is the URI to the cluster network.
+<5> `control_subnet` is the URI to the control subnet.
+<6> `image` is the URI to the {op-system} image.
+<7> `machine_type` is the machine type of the instance, for example `n1-standard-4`.
+<8> `bootstrap_ign` is the URL output when creating a signed URL above.
+
+. Create the deployment using the `gcloud` CLI:
```

Comment:
s/using/by using

> ! using n
To avoid ambiguity, replace this gerund with either “by using” or “that uses.”